packages/flutter/test/rendering/editable_test.dart

```diff
@@ -14,10 +14,7 @@ import 'rendering_tester.dart';
 
 class FakeEditableTextState with TextSelectionDelegate {
   @override
-  TextEditingValue get textEditingValue { return const TextEditingValue(); }
-
-  @override
-  set textEditingValue(TextEditingValue value) { }
```

Comment:
Why do we want to remove this setter?

Reply:
So that we're not disconnecting the text field from text model, leaving a stateless text field that prevents testing of, in this case, delete.